js/apiv2.js

```diff
@@ -109,6 +116,22 @@ const api = {
         return this.apiSession.put('/update_user_session/', sessionVarsToUpdate)
             .then(response => response.statusText)
             .catch(this.logFailure)
+    },
+    async downloadTemplate () {
+        // TODO
+            // return await this.templatesInstance.get('/')
+        console.log("API request to get Templates");
+        return await Promise.resolve({data: "EXCEL TEMPLATE"})
+            .then(response => response.data)
+            .catch(this.logFailure);
```

Comment:
Are we going to provide user feedback in some way if the api call fails?  Not sure if that is supposed to be part of this ticket or not, or if it's been speced out at all.

Reply:
Yeah I plan on updating/handling the api calls on the Generate Excel ticket #2543. We can alert users either by a view in the popover or with PNotify for api failed try again.